katalog/configs/ingress-nginx/kustomization.yaml

```diff
@@ -0,0 +1,21 @@
+# Copyright (c) 2022 SIGHUP s.r.l All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+  - output.yml
+  - flow.yml
+
+namespace: ingress-nginx
+
+generatorOptions:
+  disableNameSuffixHash: true
+
+secretGenerator:
```

Comment:
why a secret instead of a configmap? I don't see sensitive data inside the index templates, is it because the operator expects a secret or it could potentially include sensitive data?
(the same applies for the other configs, writing this just once)

Reply:
It requires a secret